auth/user_mgt.go

```diff
@@ -468,6 +469,11 @@ func IsEmailAlreadyExists(err error) bool {
 	return hasAuthErrorCode(err, emailAlreadyExists)
 }
 
+// IsEmailNotFound checks if the given error was due to the user record corresponding to the email not being found.
```

Comment:
Is this comment parsed into reference docs?  If so, we would probably want to make the literal `IsEmailNotFound` code font (not sure if backticks work for Go).

We could do that separately, since the same would apply for IsInsufficientPermission and any other similar things.

Reply:
These are visible externally. All other functions follow the same pattern (without backticks):
https://pkg.go.dev/firebase.google.com/go@v3.13.0+incompatible/auth#IsConfigurationNotFound
I am not sure if backticks work for Go either.
Will need to confirm with @hiranya911

Reply:
This is consistent with godoc conventions. Comment should start with the name of the function/type name without any formatting. See https://blog.golang.org/godoc and https://golang.org/src/go/doc/example.go

Reply:
Thanks @hiranya911